test/CMakeLists.txt

```diff
@@ -0,0 +1,91 @@
+add_subdirectory(functional/fixtures)  # compile test programs
+get_directory_property(GENERATED_HELP_TAGS DIRECTORY ${PROJECT_SOURCE_DIR}/runtime DEFINITION GENERATED_HELP_TAGS)
```

Comment:
There's no need for this hack in hindsight - this should just be a target instead.

Reply:
I spoke too soon. I'll revert this and look at it in another PR